packages/react/package.json

```diff
@@ -27,10 +27,12 @@
   },
   "dependencies": {
     "@aws-amplify/ui": "3.0.1-next.0",
+    "@aws-amplify/ui-react-v1": "npm:@aws-amplify/ui-react@1.2.9",
```

Comment:
What is this for?

Reply:
Looks like it is from resolving conflicts. I will remove it